### PR TITLE
Add employees scaffold styles

### PR DESCRIPTION
**Pull request by jestradadeveloper**
Add styles with sass for components on the employees crud views, also create
a nuevo crud layout for this crud content

# Description
Add styles with sass for components on the employees crud views, also create
a nuevo crud layout for this crud content

  ## Screenshots
Captura de Pantalla 2021-10-19 a la(s) 19 54 04

Captura de Pantalla 2021-10-19 a la(s) 19 54 23
Captura de Pantalla 2021-10-19 a la(s) 19 54 42

### CODE REVIEW

File: app/javascript/packs/components/_button.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    }
+}
+.btn{
+    &__primary{
+        @apply bg-gray-900 flex-shrink-0 h-12 text-white px-5 rounded flex items-center justify-center mx-1;
+        &:hover{
+            @apply bg-gray-600;
+        }
+    }
+}
```
Comment (YilKanda) [inline app/javascript/packs/components/_button.scss:17]: check indentation

File: app/views/application/_crud_layout.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    </div>
+    <div class="flex w-full">
+        <div class="w-4/5">
+            <%= yield :primary_content %>
+        </div>
+        <div class="w-1/5 text-center flex">
+            <%= yield :actions %>
+        </div>
+    </div>
+</div>
```
Comment (YilKanda) [inline app/views/application/_crud_layout.html.erb:13]: check indentation

File: app/views/employees/edit.html.erb [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
+<% end %>
+<%= content_for :primary_content do %>
+    
+    <%= render 'form', employee: @employee %>
+<% end %>
+<%= content_for :actions do %>
+    <%= link_to 'Show', @employee, class:'btn btn__primary' %> 
+    <%= link_to 'Back', employees_path, class:'btn btn__primary' %>
+<% end %>
+<%= render 'crud_layout' %>
```
Comment (YilKanda) [inline app/views/employees/edit.html.erb:12]: check indentation

File: app/views/employees/index.html.erb [hunk 4]
```diff
@@ -1,6 +1,9 @@
-<p id="notice"><%= notice %></p>
 
-<h1>Employees</h1>
+<%= content_for :title do %>
+    <h1 class="text-primary text-2xl text-center">Employees</h1>
```
Comment (YilKanda) [inline app/views/employees/index.html.erb:3]: check indentation

File: app/javascript/packs/components/_button.scss (hunk 1 above)
Comment (jestradadeveloper) [reply app/javascript/packs/components/_button.scss]: The indentation spaces were modified in this file

File: app/views/application/_crud_layout.html.erb (hunk 2 above)
Comment (jestradadeveloper) [reply app/views/application/_crud_layout.html.erb]: The indentation spaces were modified in this file

File: app/views/employees/index.html.erb (hunk 4 above)
Comment (jestradadeveloper) [reply app/views/employees/index.html.erb]: The indentation spaces were modified in this file

File: app/views/employees/show.html.erb [hunk 5]
```diff
@@ -1,24 +1,19 @@
...
+  </p>
+  <p>
+    <strong>Private number:</strong>
+    <%= @employee.private_number %>
+  </p>
+<% end %>
+<%= content_for :actions do %>
+  <%= link_to 'Edit', edit_employee_path(@employee) %> |
+  <%= link_to 'Back', employees_path %>
+<% end %>
```
Comment (YilKanda) [inline app/views/employees/show.html.erb:18]: Here, aren't you going to add styles?

File: app/views/employees/index.html.erb [hunk 6]
```diff
@@ -1,33 +1,37 @@
...
+          <td><%= employee.private_number %></td>
+          <td><%= link_to 'Show', employee %></td>
+          <td><%= link_to 'Edit', edit_employee_path(employee) %></td>
+          <td>
+            <%= button_to 'Delete', employee_path(employee), :method => 'delete', :confirm => 'Are you sure?' %>
+          </td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
```
Comment (YilKanda) [inline app/views/employees/index.html.erb:29]: Here, aren't you going to add styles?

Comment (jestradadeveloper) [reply app/views/employees/index.html.erb]: This is now updated with some styles

**Pull request merged**